plugin.video.catchuptvandmore/resources/lib/channels/ca/tva.py

```diff
@@ -258,19 +258,15 @@ def get_video_url(plugin,
     data_player = re.compile(
         r'data-player\=\"(.*?)\"').findall(resp.text)[0]
     data_video_id = re.compile(
-        r'data-video-id\=\"(.*?)\"').findall(resp.text)[0]
+        r'\"referenceId\":\"(.*?)\"').findall(resp.text)[0]
 
     # Method to get JSON from 'edge.api.brightcove.com'
     resp = urlquick.get(
-        URL_BRIGHTCOVE_VIDEO_JSON % (data_account, data_video_id),
+        URL_BRIGHTCOVE_VIDEO_JSON % (data_account, "ref:%s" % data_video_id),
         headers={
-            'User-Agent':
-            web_utils.get_random_ua(),
-            'Accept':
-            'application/json;pk=%s' %
-            (get_brightcove_policy_key(data_account, data_player)),
-            'X-Forwarded-For':
-            plugin.setting.get_string('header_x-forwarded-for')
+            'User-Agent': web_utils.get_random_ua(),
+            'Accept': 'application/json;pk=%s' % (get_brightcove_policy_key(data_account, data_player)),
+            'X-Forwarded-For': plugin.setting.get_string('header_x-forwarded-for')
```

Comment:
We don't want add-ons to circumvent GEO blocks, both in streaming and downloading. That means that using `x-forwarded-for` is not allowed.